<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="cbo_itens">
            <item>
             <property name="text">
              <string>Item 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Item 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Item 3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Item 4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Item 5</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_adicionar">
            <property name="text">
             <string>Adicionar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_adicionarTodos">
            <property name="text">
             <string>Adic. Todos</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="lsw_listaItem"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_transferirSelecionado">
          <property name="toolTip">
           <string>Adicionar elementos marcados</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_transferirTodos">
          <property name="toolTip">
           <string>Adicionar todos os itens da lista</string>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="lsw_listaFinal"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
